react/week1/first-react-homework/src/TodoList/todoList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import TodoItem from './todoItem'
+
+function TodoList() {
+    return <div>
+        <ul>
+            {TodoItem().map(todo => <TodoListItem id={todo.id} description={todo.description} date={todo.date} />)}
+        </ul>
```

Comment:
Just a small comment:
The console is showing a warning that says `Each child in a list should have a unique "key" prop.`

you can avoid the problem by simply providing a uniqe `key` that have the value of `todo.id`:

```suggestion
        <ul>
            {TodoItem().map(todo => <TodoListItem key={todo.id} id={todo.id} description={todo.description} date={todo.date} />)}
        </ul>
```

You will learn more about it in the next weeks :)

Reply:
Thank you, I`ll read about it and change it acordingly.